src/lib/services/user-service.ts

```diff
@@ -426,6 +442,30 @@ class UserService {
     async getUserByPersonalAccessToken(secret: string): Promise<IUser> {
         return this.store.getUserByPersonalAccessToken(secret);
     }
+
+    async updateLastSeen(): Promise<void> {
+        if (this.lastSeenSecrets.size > 0) {
+            const toStore = [...this.lastSeenSecrets];
+            this.lastSeenSecrets = new Set<string>();
+            await this.store.markSeenAt(toStore);
+        }
+
+        this.seenTimer = setTimeout(
+            async () => this.updateLastSeen(),
+            minutesToMilliseconds(3),
```

Comment:
I don't have a problem with it, just curious, why 3? It feels like such a carefully chosen number

Reply:
It's just so it's aligned with the other PR: https://github.com/Unleash/unleash/pull/2514#discussion_r1033938370

Reply:
Ah!